src/lifecycle/created.js

```diff
@@ -162,10 +162,10 @@ export default function (options) {
 
     // Native custom elements automatically inherit the prototype. We apply
     // the user defined prototype directly to the element instance if not.
-    // Note that in order to catch modified prototype chains - such as when
-    // setPrototypeOf() or ES6 classes are used - we must walk each prototype
-    // and apply each member directly.
-    if (!options.isNative) {
+    // Skate will always add lifecycle callbacks to the definition. If native
```

Comment:
This bit was causing tests to fail in browsers that support native custom elements. The use case was:

```js
var el = document.createElement('div');
el.setAttribute('is', 'custom-element');
skate.init(el);
```

This isn't natively supported by web components, but is functionality that used to work, and be supported by, Skate. This actually doesn't add any overhead, we're just checking to see if the element has been natively set up in a different, more predicable, manner.